src/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Nagłówek strony</title>
+    <title>IceCream</title>
```

Comment:
A nie powinno być "Ice cream made with passion" ?

Reply:
Ok, myślę, że wystarczyłby tytuł "IceCream", ale jeżeli grupa zadecyduje to zmienimy na inny np. "Ice Cream - made with passion".

Reply:
Dobra, zmienię na "Ice cream made with passion"